<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureDlg</class>
 <widget class="QDialog" name="ConfigureDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnOk">
       <property name="text">
        <string>Ok</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="m_pTabWidget1">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab1">
      <attribute name="title">
       <string>Editor</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <item row="9" column="0">
        <widget class="QLabel" name="m_pLabelCRLFMode">
         <property name="text">
          <string>CR/LF Modus:</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="m_pTabSpaceLabel">
         <property name="text">
          <string>Tabspace:</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QLineEdit" name="m_pTabSpace">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <spacer name="Spacer2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="6" column="0" colspan="3">
        <widget class="QCheckBox" name="m_pAutoIndent">
         <property name="text">
          <string>Autoindent</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QCheckBox" name="m_pSpacesForTabs">
         <property name="text">
          <string>Insert spaces for tabs</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="3">
        <widget class="QCheckBox" name="m_pShowWhitespaces">
         <property name="text">
          <string>Show whitespaces</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QComboBox" name="m_pCRLFModus">
         <item>
          <property name="text">
           <string>Windows</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Unix</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Mac</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="9" column="2">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="0" colspan="3">
        <widget class="QCheckBox" name="m_pShowLineNumbers">
         <property name="text">
          <string>Show line numbers</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="3">
        <widget class="QCheckBox" name="m_pBraceMatching">
         <property name="text">
          <string>Brace matching</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="3">
        <widget class="QCheckBox" name="m_pCodeFolding">
         <property name="text">
          <string>Code folding</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Icon size [Pixel]:</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QComboBox" name="m_pIconSize">
         <item>
          <property name="text">
           <string>14</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>16</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>24</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>32</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>48</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>64</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>92</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>128</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>192</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>256</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>512</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0" colspan="3">
        <widget class="QCheckBox" name="m_pShowCurrentLine">
         <property name="text">
          <string>Highlight current line</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab2">
      <attribute name="title">
       <string>Script</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <item row="0" column="1">
        <widget class="QLineEdit" name="m_pScriptCmd">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="4">
        <widget class="QCheckBox" name="m_pAutoSave">
         <property name="text">
          <string>Auto save before execution</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="btnAssignExtension">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>assign a file extension for this interpreter</string>
         </property>
         <property name="text">
          <string>*.???</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="4">
        <widget class="QCheckBox" name="m_pMeasureExecutionTime">
         <property name="text">
          <string>Measure execution time</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="4">
        <widget class="QCheckBox" name="m_pInteractionModus">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Interactive Modus</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="m_pScriptArgumentsLabel">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Arguments:</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="4">
        <widget class="QCheckBox" name="m_pChangeActDir">
         <property name="text">
          <string>Change act. directory before execution</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="m_pUseUserInterpreter">
         <property name="toolTip">
          <string>enter a new user interpreter</string>
         </property>
         <property name="text">
          <string>Run</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0" colspan="4">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="0" colspan="4">
        <widget class="QCheckBox" name="m_pPipeScriptToInterpreter">
         <property name="text">
          <string>Pipe script to interperter</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="3">
        <widget class="QComboBox" name="m_pScriptArguments">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="4">
        <widget class="QComboBox" name="m_pScriptCmdList">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text">
           <string>minscript</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>perl</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>python</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>python -i</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ruby</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>tclsh</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>lua</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>scheme48</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>runhugs</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="8" column="0" colspan="4">
        <widget class="QCheckBox" name="m_pShowOutputArea">
         <property name="text">
          <string>Enable script execution</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="btnSelectCommand">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>select an interpreter</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0" colspan="4">
        <widget class="QCheckBox" name="m_pUseRootForFileDialog">
         <property name="text">
          <string>Use / as SD card path</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab3">
      <attribute name="title">
       <string>Misc</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <item row="8" column="0">
        <widget class="QLabel" name="m_pAutoCompletionLabel">
         <property name="text">
          <string>Auto completion file:</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="m_pAutoCompletThreshLabel">
         <property name="text">
          <string>Threshold:</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <spacer name="Spacer3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>59</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="m_pOutputRatioLabel">
         <property name="text">
          <string>Output ratio in %:</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="8" column="1" colspan="2">
        <widget class="QLineEdit" name="m_pAutoCompletion">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="9" column="1" colspan="2">
        <widget class="QLineEdit" name="m_pAutoCompleteThresh">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="3">
        <widget class="QCheckBox" name="m_pAutoClearOutput">
         <property name="text">
          <string>automatic clear of output</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="3">
        <widget class="QCheckBox" name="m_pAutoFileLoad">
         <property name="text">
          <string>automatic load at startup</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="3">
        <widget class="QCheckBox" name="m_pShowFileDialog">
         <property name="text">
          <string>show file dialog at startup</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="3">
        <widget class="QCheckBox" name="m_pCloseButtonInTabHeader">
         <property name="text">
          <string>Closebutton in tabheader</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="3">
        <widget class="QCheckBox" name="m_pUseToolbar">
         <property name="text">
          <string>Use toolbar</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QCheckBox" name="m_pUseNewIcons">
         <property name="text">
          <string>Use new icons</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="3">
        <widget class="QCheckBox" name="m_pFlagUtf8">
         <property name="text">
          <string>UTF-8 encoding</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1" colspan="2">
        <widget class="QSpinBox" name="m_pOutputRatio">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>30</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>m_pTabSpace</tabstop>
  <tabstop>m_pSpacesForTabs</tabstop>
  <tabstop>m_pShowWhitespaces</tabstop>
  <tabstop>m_pShowLineNumbers</tabstop>
  <tabstop>m_pCodeFolding</tabstop>
  <tabstop>m_pAutoIndent</tabstop>
  <tabstop>m_pBraceMatching</tabstop>
  <tabstop>m_pInteractionModus</tabstop>
  <tabstop>m_pChangeActDir</tabstop>
  <tabstop>m_pAutoSave</tabstop>
  <tabstop>m_pFlagUtf8</tabstop>
  <tabstop>m_pShowFileDialog</tabstop>
  <tabstop>m_pOutputRatio</tabstop>
  <tabstop>m_pAutoClearOutput</tabstop>
  <tabstop>m_pAutoCompleteThresh</tabstop>
  <tabstop>m_pTabWidget1</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>m_pScriptCmdList</sender>
   <signal>activated(QString)</signal>
   <receiver>m_pScriptCmd</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>208</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnOk</sender>
   <signal>clicked()</signal>
   <receiver>ConfigureDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>313</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>345</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>ConfigureDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>133</x>
     <y>313</y>
    </hint>
    <hint type="destinationlabel">
     <x>344</x>
     <y>340</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSelectCommand</sender>
   <signal>clicked()</signal>
   <receiver>ConfigureDlg</receiver>
   <slot>sltSelectCommand()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>289</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>25</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pUseUserInterpreter</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_pScriptCmd</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>208</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pUseUserInterpreter</sender>
   <signal>toggled(bool)</signal>
   <receiver>btnSelectCommand</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>289</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pUseUserInterpreter</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_pScriptCmdList</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>123</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pUseUserInterpreter</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_pScriptArguments</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pUseUserInterpreter</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_pScriptArgumentsLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>69</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pUseUserInterpreter</sender>
   <signal>toggled(bool)</signal>
   <receiver>btnAssignExtension</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>35</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>370</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnAssignExtension</sender>
   <signal>clicked()</signal>
   <receiver>ConfigureDlg</receiver>
   <slot>sltAssignExtension()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>370</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>344</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pScriptCmd</sender>
   <signal>textChanged(QString)</signal>
   <receiver>ConfigureDlg</receiver>
   <slot>sltScriptCmdChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>208</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>4</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pScriptCmdList</sender>
   <signal>activated(QString)</signal>
   <receiver>m_pScriptArguments</receiver>
   <slot>clearEditText()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>163</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>sltSelectCommand()</slot>
  <slot>sltAssignExtension()</slot>
  <slot>sltScriptCmdChanged(QString)</slot>
 </slots>
</ui>
